pulp_rpm/app/migrations/0017_merge_advisory_collections.py

```diff
@@ -0,0 +1,53 @@
+from django.db import (
+    migrations,
+    models,
+    )
+
+
+def do_clone(apps, advisory, in_collection):
+    # We're cloning collection - find its updatecollectionpackages
+    uc_packages = in_collection.packages.all().iterator()
```

Comment:
I would suggest moving ```.iterator()``` to the loop itself,  it makes it more clear what the behavior of this is.

Reply:
done